### PR TITLE
[thread-cert] refactor case 9.2.10 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_9_2_10_PendingPartition.py [hunk 1]
```diff
@@ -176,6 +171,77 @@ def test(self):
...
+        # Step 12: Router MUST send a unicast MLE Data Response to SED_1
+        _rpkts.filter_wpan_dst64(SED).filter_mle_cmd(MLE_DATA_RESPONSE).must_next().must_verify(lambda p: {
+            SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, NETWORK_DATA_TLV, ACTIVE_TIMESTAMP_TLV, PENDING_OPERATION_DATASET_TLV
+        } <= set(p.mle.tlv.type) and {
+            NM_CHANNEL_TLV, NM_NETWORK_MESH_LOCAL_PREFIX_TLV, NM_PAN_ID_TLV, NM_DELAY_TIMER_TLV,
+            NM_ACTIVE_TIMESTAMP_TLV
+        } <= set(p.thread_meshcop.tlv.type))
+
+        # Step 14: After NETWORK_ID_TIMEOUT, Router MUST start a new partition
+        _rpkts.filter_ipv6_dst(LINK_LOCAL_ALL_ROUTERS_MULTICAST_ADDRESS).filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_10_PendingPartition.py:231]: How to verify it's after`NETWORK_ID_TIMEOUT`  ?

Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_10_PendingPartition.py:231]: isolate Router_1, MED_1, and SED_1 for 300s > 120s, Router_1 will start a new partition

Comment (simonlingoogle) [reply tests/scripts/thread-cert/Cert_9_2_10_PendingPartition.py:231]: I mean beside the test procedure, we should also verify the time gap in the verify?

Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_10_PendingPartition.py]: verified the packets time interval > 300 s

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Review by jwhui — APPROVED** [review]

**Pull request merged**